examples/pendulum/pendulum.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 
 from pilco.environments import Environment
-from pilco.policies import RBFPolicy, TransformedPolicy
+from pilco.policies import PendulumEQPolicy, TransformedPolicy
```

Comment:
Yeah, as you said, it's probably cleaner to move PendulumEQPolicy to `examples/`. I think we should keep the contents of the package general.